src/CalendarAvailability.vue

```diff
@@ -100,6 +100,10 @@ export default {
 			type: String,
 			required: true,
 		},
+		l10nListLabel: {
+			type: String,
+			required: true,
+		},
```

Comment:
This doesn't seem obvious to a component's user, what is `listLabel` here. Same as `l10nMonday` I'd call it in a form of `l10nENGLISH` or provide a default value with English string.

Reply:
If this prop is required, then it is a breaking change

Reply:
Agreed, I wanted users of the library follow best practices with ARIA, but using a default would capture both best practices and not be a breaking change. Thank you :)